### ISSUE TITLE
Map and road styles comparison

### ISSUE THREAD

**Issue by breversa** [osmandapp/Osmand#1668]
Is there any document describing the different features/pros/cons of each map style (especially OsmAnd, Touring view, UniRS, LightRS. Nautical and ski are self-explanatory) and route style (default, orange, german road atlas, american road atlas, high contrast roads) ?

In particular, I'd like to know :
- what style has been updated and when (I guess some styles are older/obsolete than others)
- what is the intended purpose/application of each style (what's the different between LightRS and UniRS ?)
- are some styles faster to display than others (my device is getting old) ?
- any contact address to the developer ?

**Comment by vshcherb** [comment]
This is better to ask on forum, but let's see if somebody replies to it here.

**Comment by breversa** [comment]
Wait, there's an OsmAnd forum ? Where ??

**Comment by vshcherb** [comment]
Google groups on website

**Comment by breversa** [comment]
Damn, I found it (thanks anyway ! :-) ), but the osmand.net website could REALLY use improvement here : no link or mention of the forum/github in the FAQ ; you need to scroll all the way down to see "External links" in the page footer. :-(

**Comment by sonora** [comment]
The OsmAnd groupd is netionend at the botto of our web site, under "Extenal links", but maybe thisis too well hidden ... :-)

We do not have a good legend or consolidated description for the map styles. A legend for the default (OsmAnd) style is included as an online link in the Help function of the map. Most other styles depend on that style, and only have some deviations, but server certain purposes.

For the Touring view, the philosophy is summarized in the comment section of the file itself, as follows (I agree our documentation has room for improvement, but it is not a priority ... :-) ):

**Comment by breversa** [comment]
> The OsmAnd groupd is netionend at the botto of our web site, under "Extenal links", but maybe thisis too well hidden ... :-)

Indeed : I was desperately looking for a "Community" or "Contribute" page, I explored the FAQ (where the single link to github is for voice), to no avail. Only when I explored the farthest corner of the page (yup, bottom right-hand corner… Couldn't be further !) did I find the link. So yeah, I guess it's too well hidden. :-)

Anyway, thank you for your reply ! Is there any place to mention it ? In the website FAQ, maybe ?
